buildbot-conf/private.py.sample

```diff
@@ -0,0 +1,11 @@
+pbcsUser    = 'password'
+pbcsPwd     = 'password'
+webAuthUser = 'password'
+webAuthPwd  = 'password'
+
+slave34Pwd = 'password'
```

Comment:
This file is a sample and doesn't affect already-deployed files. Sample passwords shouldn't be used in production, so it's irrelevant what they are in the sample.